src/components/SwapForm/SwapFormToSections/ToSectionCarbonOffset.tsx

```diff
@@ -50,14 +52,14 @@ export const ToSectionCarbonOffset = ({
           { xs: 8, sm: 16 },
         ]}>
         <Col span={10}>
-          <div className="form-text">To retire</div>
+          <div className="form-text">You pay</div>
```

Comment:
Is it on purpose that the wording changed?

Reply:
Yeah this is on purpose. instead of inputting an amount of money and calculating how many tonnes you retire, you now input a desired amount of carbon you want to retire. The calculated amount by the route
 is now the price you pay for that.